tests/sandbox/test_429_errors.py

```diff
@@ -20,8 +20,6 @@ def test_too_many_requests_get(nhsd_apim_proxy_url, correlation_id, method):
         "X-Correlation-Id": correlation_id
     })
 
-    error_handler.handle_504_retry(resp)
```

Comment:
This is already done as part of `Assertions.assert_error_with_optional_correlation_id`.